Custom domain: update authz server task topic (#4213)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

https://<id.domain.name>/oauth2/default/.well-known/openid-configuration
```

You might notice that the URL has your Okta `dev-*` domain, rather than your custom domain name.

To fix this, update your authorization server to use your custom domain:

1. Sign in to your Okta account and go to **Security** > **API** > **Authorization Servers**.
1. Locate the  **default** custom authorization server, and then click **Edit**.
1. In the **Settings** section, click **Edit**.

> 1. Change the **Issuer** to **Custom URL** or **Dynamic (based on request domain)** (if enabled).

Reviewer: Can we just have the recommendation be Dynamic, rather than mentioning the Custom URL option? In general Dynamic should be the guidance we give moving forward.

Reviewer: Also was the numbering change intentional?

Author: Sure, I can make that change no problem.

The numbering change is intentional. VuePress renders the `1s` as an ordered list. It helps avoid mis-numbering that results from manual errors.

> 1. Try `./well-known/openid-configuration` again. It should display your custom domain.

Reviewer 2: ```suggestion
1. Make a request to `./well-known/openid-configuration` again. Your custom domain is returned.
```
or "Your custom domain appears."

Author: Thank you :)
